scripts/Tools/xmlquery

```diff
@@ -330,10 +330,11 @@ def _main_func(description):
         wrapper=textwrap.TextWrapper()
         wrapper.subsequent_indent = "\t\t\t"
         wrapper.fix_sentence_endings = True
+
+    cnt = 0
```

Comment:
This is a bugfix for xmlquery. The output was wrong with --value when multiple values were requested (comma was in the wrong place).